app/services/katello/pulp3/migration.rb

```diff
@@ -6,6 +6,13 @@ class Migration
       attr_accessor :smart_proxy
       GET_QUERY_ID_LENGTH = 90
 
+      CONTENT_TYPES = [
+        Katello::FileUnit,
+        Katello::DockerManifestList,
+        Katello::DockerManifest,
+        Katello::DockerTag
+      ].freeze
```

Comment:
i'd really like there to be a single 'definition' for the content types in pulp we are migrating.  Currently the full migration rake task defines it here:  
https://github.com/Katello/katello/blob/master/lib/katello/tasks/pulp3_migration.rake#L4

but i think it makes sense to move that here, add docker, and then derive the content types from the repository like this is doing:

https://github.com/Katello/katello/blob/master/app/services/katello/pulp3/migration.rb#L32

That way, all of these migration tasks have a single 'source' for what should be migrated